WIP global rephrasing (#222)
meilisearch/documentation: README.md

# MeiliSearch Documentation

This repository is the documentation for the **open-source search engine API [MeiliSearch](https://github.com/meilisearch/meilisearch)**.

## Contributing

Hey, thank you so much for wanting to contribute! If you'd like to help improve this documentation, we'd love to have you! If you think something is missing or could be improved, issues are highly appreciated.

To contribute, clone this repository, commit your changes and open a pull request.

> Thank you for considering helping out!

Reviewer: Redundant with the `Hey, thank you so much for wanting to contribute!` 
```suggestion
```

### Testing

A complete test can be done using the following command:

```
$ yarn test
```

> This will also trigger `yarn build` on any pull request to master.

Reviewer: This changes the meaning of the sentence to another sentence. I wanted to say, the tests are triggered on build and on any pull request to master.
For a little explanation, the documentation is build, meaning it transpiles our javascript and vue files into a browser-readable-javascript. Before it does this transpilation it will first pass the tests so that it does not build if the tests faies. 
```suggestion
The tests are triggered on build and on any pull request to master.
```

Author: I get it! Yes you're right
